frontend/src/concepts/pipelines/content/tables/pipeline/PipelinesTableRow.tsx

```diff
@@ -66,7 +66,7 @@ const PipelinesTableRow: React.FC<PipelinesTableRowProps> = ({
   return (
     <>
       <Tbody isExpanded={isExpanded}>
-        <Tr>
+        <Tr data-testid={`pipeline-row ${pipeline.pipeline_id}`}>
```

Comment:
test IDs typically do not have whitespace and should be kebab-case.

Reply:
We support white space separated test IDs as individual queryable IDs.
`cy.findByTestId(['firstID', 'secondID'])`
Similarly we can also query for just a single test id: `cy.findByTestId(['firstID'])`

This support is similar to query selectors: `document.querySelector('[data-testid~="firstID"]')`

IMO separating by white space provides more flexibility in case you want to find all rows or a specific row.

Reply:
I see, I didn't realize we had overwritten the default findByTestId function to allow for this. As long as it's being used in that way here, that makes sense.